setup.cfg

```diff
@@ -13,8 +13,8 @@ author = Greg Hellings
 author_email = greg.hellings@gmail.com
 maintainer = Greg Hellings
 maintainer_email = greg.hellings@gmail.com
-license = GPLv3
-; license_file = LICENSE
+license = MIT
+license_file = LICENSE
```

Comment:
If it remember well, these are exclusive. Keep line 17 commented or we will fail to release on pypi.